scripts/detect-package-json-changes.sh

```diff
@@ -1,14 +1,9 @@
 #!/bin/bash
 
-packageJsonStatus="$(git diff --name-status develop -- package.json)"
-packageLockJsonStatus="$(git diff --name-status develop -- package-lock.json)"
+licenseFileStatus="$(diff CIRCLE-THIRD-PARTY-LICENSES.csv THIRD-PARTY-LICENSES.csv)"
```

Comment:
thought: might be able to do a git diff to avoid the extra file too, but might not be worth the effort to change it